docker/dockerbuild/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/python:2.7.17-stretch
```

Comment:
it doesnt contain

i changed that because of issues with openssl

using the 3.10 image, would running the following be sufficient ?
`pyenv install 2.7.18 && pyenv versions | grep 2.7.18`

Reply:
I think it can work

Reply:
done